jest.config.js

```diff
@@ -1,16 +0,0 @@
-// For a detailed explanation regarding each configuration property, visit:
```

Comment:
Why is this file deleted?

Reply:
Jest configurations can be specified either in the package.json file, or a jest.config.js file. Since we already have settings for it in the package.json file, it was redundant to have this file so I deleted it.